totp/__init__.py

```diff
@@ -114,11 +114,17 @@ def copy_to_clipboard(text):
         elif platform.system() == "Windows":
             command = ["clip"]
         else:
-            selection = os.environ.get(
+            clipboard_type = os.environ.get(
                 "PASSWORD_STORE_X_SELECTION",
                 "clipboard",
             )
-            command = ["xclip", "-selection", selection]
+            session_type = os.environ.get("XDG_SESSION_TYPE")
+            if session_type == "wayland":
```

Comment:
It's probably safer to do:

```suggestion
            if "WAYLAND_DISPLAY" in os.environ:
```

The `WAYLAND_DISPLAY` must be set if running Wayland, but XDG_SESSION_TYPE is not mandatory and might be absent.

Reply:
Good idea, applied!